doc/api/v0.yaml

```diff
@@ -124,6 +124,24 @@ paths:
         '404':
           description: Account with given ID was not found
 
+  '/api/v0/block/block0':
+    get:
+      description: Gets block0
+      operationId: Block0
+      tags:
+        - block
+      responses:
+        '200':
+          description: Success
+          content:
+            application/octet-stream:
+              schema:
+                description: Binary blob with block0
+                type: string
+                format: binary
+        '400':
+          description: Block was not found
+
```

Comment:
how is it different from the `/api/v0/block/{block_id}` ? also the `block_id`  of the block0 is already available in the settings: `/api/v0/settings`.

Reply:
> how is it different from the /api/v0/block/{block_id} ?

The result is the same. It just fetches directly the block0 data without the need to through the settings.

> also the block_id of the block0 is already available in the settings: /api/v0/settings.

Yes, also mentioned in the comment.